app/views/layouts/vueapp.html.erb

```diff
@@ -50,5 +50,8 @@
     <div id="app"></div>
     <noscript id="noscript">This app works best with JavaScript enabled.</noscript>
     <%= yield %>
+    <audio autoplay loop muted id="notif-audio">
+      <source src="<%= audio_path 'ding.mp3' %>" type="audio/mp3">
```

Comment:
Why can't we use the existing audio file?

Reply:
Needs to be in assets folder to include in `.erb` files. @sojan-official Any idea to use the same file in `app/javascript/shared/assets/audio/ding.mp3` in `.erb` files?

Reply:
Please move it to the public folder in this case.

Reply:
Use an .ogg file not mp3